src/components/value-chart/ExtremeLabels.js

```diff
@@ -78,7 +84,7 @@ export default function Labels({ color, width }) {
           }}
           width={width}
         >
-          {formatUSD(smallestY.y)}
+          {formatNative(smallestY.y, null, nativeSelected)}
```

Comment:
just a note to pass along nativeCurrency if you change to use the utility function in the formatNative function